From 6f3ff0be4ca474c54b4facac99362b9b02b37b25 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Sun, 14 Apr 2013 20:25:46 +0530
Subject: [PATCH 067/124] fsl/usb: Turn on cache snooping hardware for 64-bit
 Kernel

Turn on cache snooping hardware for 64-bit Kernel

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Change-Id: I1f59eb6a53fd420cda87873bec0ec82094a022de
Reviewed-on: http://git.am.freescale.net:8181/1320
Reviewed-by: Mehresh Ramneek-B31383 <ramneek.mehresh@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index 5ab5657..73f04a8 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -338,7 +338,7 @@ static int dr_controller_setup(struct fsl_udc *udc)
 	}
 #endif
 
-#if defined(CONFIG_PPC32) && !defined(CONFIG_NOT_COHERENT_CACHE)
+#if !defined(CONFIG_NOT_COHERENT_CACHE)
 	/* Turn on cache snooping hardware, since some PowerPC platforms
 	 * wholly rely on hardware to deal with cache coherent. */
 
-- 
2.1.0.27.g96db324

